### PR TITLE
Moved network_proxy under the endpoint object hierarchy.

### PR DESCRIPTION
**Pull request by pagbabian-splunk**

### CODE REVIEW

File: objects/entity/endpoint/network_proxy.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "caption": "Network Proxy Endpoint",
+  "name": "network_proxy",
+  "description": "The network proxy endpoint object describes a network proxy. Defined by D3FEND <a target='_blank' href='https://d3fend.mitre.org/dao/artifact/d3f:ProxyServer/'>d3f:ProxyServer</a>.",
+  "extends": "network_endpoint",
```
Comment (floydtree) [inline objects/entity/endpoint/network_proxy.json:5]: Wasn't the plan to extend `endpoint`?  The folder hierarchy will not be consisent if you extend `network_endpoint` - all the objects that extend endpoint are in this folder

**Review by floydtree — CHANGES_REQUESTED** [review]

File: objects/entity/endpoint/network_proxy.json (hunk 1 above)
Comment (pagbabian-splunk) [reply objects/entity/endpoint/network_proxy.json:5]: network_endpoint extends endpoint.

Comment (floydtree) [reply objects/entity/endpoint/network_proxy.json:5]: Will be discussed further in the weekly System activity call

**Review by rroupski — APPROVED** [review]

**Review by floydtree — APPROVED** [review]

**Review by Aniak5 — APPROVED** [review]
LGTM

**Pull request merged**